packages/main/src/YearPicker.ts

```diff
@@ -76,21 +76,18 @@ type YearPickerNavigateEventDetail = {
 })
 /**
  * Fired when the user selects a year via "Space", "Enter" or click.
- * @public
  */
 @event("change")
 /**
  * Fired when the timestamp changes - the user navigates with the keyboard or clicks with the mouse.
  * @since 1.0.0-rc.9
```

Comment:
If we are removing `@since 2.2.0` tags, should we remove those(all) `@since` tags aswell ?

Reply:
it was removed by mistake, it's good to have them as tags for traceability